@types/@ledgerhq/hw-transport-u2f/index.d.ts

```diff
@@ -0,0 +1,18 @@
+declare module '@ledgerhq/hw-transport-u2f';
```

Comment:
Move these two to @neo-one/types and add them to the list of libraries to add to @types